sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CopyAuthorization.java

```diff
@@ -0,0 +1,197 @@
+
+import java.io.IOException;
+
+/**
+ * The CopyAuthorization model.
+ */
+@Fluent
+public final class CopyAuthorization {
+    private static final SerializerAdapter SERIALIZER = new JacksonAdapter();
+
+    CopyAuthorization() {
+        super();
+    }
+
+    /*
+     * Model identifier.
+     */
+    private String modelId;
+
+    /*
+     * Token claim used to authorize the request.
+     */
+    private String accessToken;
+
+    /*
+     * Resource Identifier.
+     */
+    private String resourceId;
+
+    /*
+     * Region of the resource.
+     */
+    private String region;
+
+    /*
+     * The time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     */
+    private long expiresOn;
+
+    /**
+     * Create a CopyAuthorization object
+     *
+     * @param modelId The model identifier
+     * @param accessToken The token used to authorize the request
+     * @param resourceId The resource identifier
+     * @param region The region of the resource
+     * @param expiresOn The expiry time of the token
+     */
+    public CopyAuthorization(final String modelId, final String accessToken, final String resourceId,
+        final String region, final long expiresOn) {
+        this.modelId = modelId;
+        this.accessToken = accessToken;
+        this.resourceId = resourceId;
+        this.region = region;
+        this.expiresOn = expiresOn;
+    }
+
+    /**
+     * Get the modelId property.
+     *
+     * @return the modelId value.
+     */
+    public String getModelId() {
+        return this.modelId;
+    }
+
+    /**
+     * Get the token claim used to authorize the request.
+     *
+     * @return the accessToken value.
+     */
+    public String getAccessToken() {
+        return this.accessToken;
+    }
+
+    /**
+     * Get the time when the access token expires. The date is represented as the
+     * number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time.
+     *
+     * @return the expirationDateTimeTicks value.
+     */
+    public long getExpiresOn() {
+        return this.expiresOn;
+    }
+
+    /**
+     * Get the Azure Resource Id of the target Form Recognizer resource
+     * where the model will be copied to.
+     *
+     * @return the resourceId value.
+     */
+    public String getResourceId() {
+        return resourceId;
+    }
+
+    /**
+     * Get the location of the target Form Recognizer resource.
+     *
+     * @return the resourceRegion value.
+     */
+    public String getRegion() {
+        return region;
+    }
+
+    /**
+     * Converts the CopyAuthorization object to its equivalent json string representation.
+     *
+     * @return the json string representation of the CopyAuthorization object.
+     * @throws IOException exception from serialization
+     */
+    public String toJson() throws IOException {
+        return SERIALIZER.serialize(this, SerializerEncoding.JSON);
```

Comment:
I don't see where we are making the changes so that when we create the json string, we transform the property names to what the REST API is expecting. i.e. the string should look something like:
```
{"modelId": "26f7c09b-3e1d-4348-b98d-18187afcbf17", "accessToken": "82ea8d63-1eaa-498c-bd8b-xxxxxxxx", "expirationDateTimeTicks": 637251564427472980, "resourceId": "xxxxx", "resourceRegion": "westus2"}
```

Reply:
without `xxx` :P